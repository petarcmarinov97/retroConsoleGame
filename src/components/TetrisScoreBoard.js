import React, { Component } from 'react';
import "../styles/ScoreBoard.css";

export default class TetrisScoreBoard extends Component {
    render() {
        const canvasHeight = "80px";
        const canvasWidth = "80px";

        return <div className="scoreboard">
            <div >
                Points
                <span id="points">0</span>
            </div>
            <div >
                Cleans
                <span id="cleans">0</span>
            </div>
            <div >
                Level
                <span id="level">0</span>
            </div>
            <div>Next</div>
            <canvas id="next" height={canvasHeight} width={canvasWidth} />
        </div>;
    }
}
