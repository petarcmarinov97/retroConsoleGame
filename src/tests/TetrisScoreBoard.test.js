import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from "enzyme-to-json";
import TetrisScoreBoard from '../components/TetrisScoreBoard';

describe('TetrisScoreBoard tests', () => {

    let wrapper;

    beforeEach(() => {
        wrapper = shallow(
            <TetrisScoreBoard />);
    })

    describe('Snapshots', () => {
        it("should render correctly", () => {
            expect(shallowToJson(wrapper)).toMatchSnapshot();
        })
    })
})