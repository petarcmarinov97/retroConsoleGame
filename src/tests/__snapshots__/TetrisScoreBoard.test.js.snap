// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TetrisScoreBoard tests Snapshots should render correctly 1`] = `
<div
  className="scoreboard"
>
  <div>
    Points
    <span
      id="points"
    >
      0
    </span>
  </div>
  <div>
    Cleans
    <span
      id="cleans"
    >
      0
    </span>
  </div>
  <div>
    Level
    <span
      id="level"
    >
      0
    </span>
  </div>
  <div>
    Next
  </div>
  <canvas
    height="80px"
    id="next"
    width="80px"
  />
</div>
`;
